sdk/src/metrics/aggregation/histogram_aggregation.cc

```diff
@@ -1,11 +1,13 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#include <memory>
```

Comment:
nit - this header can be within ENABLE_METRICS_PREVIEW macro.

Reply:
thanks moved under macro